nbs/models.hint.ipynb

```diff
@@ -17,18 +17,27 @@
    ]
```

Comment:
<p>Maybe a add a liitle summary of the example so the reader know what to expect. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>